qemu/tests/qemu_guest_agent.py

```diff
@@ -102,49 +99,45 @@ def _cleanup_open_session(self):
         except Exception:
             pass
 
-    @error.context_aware
     def gagent_install(self, params, vm, *args):
         if args and isinstance(args, tuple):
             gagent_install_cmd = args[0]
         else:
-            raise error.TestError("Missing config 'gagent_install_cmd'")
+            self.test.error("Missing config 'gagent_install_cmd'")
 
         if not gagent_install_cmd:
             return
 
-        error.context("Try to install 'qemu-guest-agent' package.",
-                      logging.info)
+        logging.info("Try to install 'qemu-guest-agent' package.")
         session = self._get_session(params, vm)
         s, o = session.cmd_status_output(gagent_install_cmd)
         if s:
-            raise error.TestFail("Could not install qemu-guest-agent package"
-                                 " in VM '%s', detail: '%s'" % (vm.name, o))
+            self.test.fail("Could not install qemu-guest-agent package in VM '%s'",
```

Comment:
I think you should have no `,` in the end of this line.

Reply:
@vivianQizhu  Thanks,  I have updated it.